src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -40,17 +41,20 @@ internal class SonarLintDaemon : ISonarLintDaemon
         private static readonly string DAEMON_HOST = "localhost";
         private static readonly int DEFAULT_DAEMON_PORT = 8050;
 
-        public const string daemonVersion = "4.0.2.2142";
-        private const string uriFormat = "http://repo1.maven.org/maven2/org/sonarsource/sonarlint/core/sonarlint-daemon/{0}/sonarlint-daemon-{0}-windows.zip";
+        public const string DefaultDaemonVersion = "4.0.2.2142";
+        public const string SonarLint_DownloadUrl_EnvVar = "SONARLINT_DAEMON_DOWNLOAD_URL";
+        public static readonly string DefaultMavenUrl = string.Format("http://repo1.maven.org/maven2/org/sonarsource/sonarlint/core/sonarlint-daemon/{0}/sonarlint-daemon-{0}-windows.zip",
```

Comment:
nitpick: this is the default version for `SonarLint_DownloadUrl_EnvVar`, so it might be better to have a name like `DefaultSonarLintDownloadUrl` ?